customTests/azure_nccl_allreduce.nhc

```diff
@@ -65,6 +65,6 @@ function check_nccl_allreduce() {
       fi
    done
 
-   die 1 "$FUNCNAME: NCCL allreduce, BUS BW (expected >=$EXP_NCCL_ALLREDUCE_BW GB/s, but measured $avg_bus_bw GB/s)"
+   die 1 "$FUNCNAME: NCCL allreduce, BUS BW (expected >=$EXP_NCCL_ALLREDUCE_BW GB/s, but measured $avg_bus_bw GB/s). FaultCode: NHC2003"
```

Comment:
same here.

Reply:
I will change it to the IBPerformance fault code

Reply:
ManualInvestigation